single-line-diagram-core/src/main/java/com/powsybl/sld/coordinate/Coord.java

```diff
@@ -4,7 +4,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.powsybl.sld.model;
```

Comment:
I think that it the coordinate package and its classes should be a sub-package of the model package, as they are part of the data model.